events/code_review_events/workflow.py

```diff
@@ -39,14 +40,30 @@
 PULSE_TASK_COMPLETED = "exchange/taskcluster-queue/v1/task-completed"
 PULSE_TASK_FAILED = "exchange/taskcluster-queue/v1/task-failed"
 
+LANDO_WARNING_MESSAGE = "Static analysis and linting is still in progress."
```

Comment:
Should it be
```suggestion
LANDO_WARNING_MESSAGE = "Static analysis and linting are still in progress."
```
?

I'm not sure which one is grammatically correct